Snake/Snake/UILayer.cs

```diff
@@ -29,8 +29,9 @@ static void Main(string[] args)
                 int score = Score.getScore(); // Get score from business layer
                 drawScore(score);
 
-
-                ConsoleKeyInfo keyInfo = Console.ReadKey();
+                // Check if arrow keys/q key pressed. If not, continue game. 
+                // Game step is 100 milliseconds 

```

Comment:
so make a constant that is called GameStepMilliseconds or GameStepTime and use that then you don't need this comment and you don't have a magic number